<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PvCWidget</class>
 <widget class="QWidget" name="PvCWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MenuWidget</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(0, 0, 0);
border: 0;</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <spacer name="verticalSpacer_6">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item alignment="Qt::AlignHCenter">
      <widget class="QLabel" name="label_PvCPlayer">
       <property name="styleSheet">
        <string notr="true">font: 75 14pt &quot;Verdana&quot;;
color: #FFF;
margin-bottom: 6px;</string>
       </property>
       <property name="text">
        <string>Name Spieler</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_PvCPlayer">
       <property name="focusPolicy">
        <enum>Qt::ClickFocus</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: #636363;
font: 75 12pt &quot;Verdana&quot;;
color: #FFF;
height: 50px;
min-width: 300px;
border-radius: 5px;</string>
       </property>
       <property name="text">
        <string>Player</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label_PvCSize">
       <property name="styleSheet">
        <string notr="true">font: 75 14pt &quot;Verdana&quot;;
color: #FFF;
margin-bottom: 6px;</string>
       </property>
       <property name="text">
        <string>Spielfeldbreite</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox_PvCFieldsize">
       <property name="focusPolicy">
        <enum>Qt::WheelFocus</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: #636363;
font: 75 12pt &quot;Verdana&quot;;
color: #FFF;
height: 35px;
padding: 0px;</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_3">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item alignment="Qt::AlignHCenter">
      <widget class="QLabel" name="label_PvCPossMoves">
       <property name="styleSheet">
        <string notr="true">font: 75 14pt &quot;Verdana&quot;;
color: #FFF;
margin-bottom: 6px;</string>
       </property>
       <property name="text">
        <string>Mögliche Züge anzeigen</string>
       </property>
      </widget>
     </item>
     <item alignment="Qt::AlignHCenter">
      <widget class="QCheckBox" name="checkBox_PvCshowPossMoves">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="focusPolicy">
        <enum>Qt::ClickFocus</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 75 11pt &quot;Verdana&quot;;
color: #FFF;</string>
       </property>
       <property name="text">
        <string>aktiviert</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_4">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item alignment="Qt::AlignHCenter">
      <widget class="QLabel" name="label_PvCDifficulty">
       <property name="styleSheet">
        <string notr="true">font: 75 14pt &quot;Verdana&quot;;
color: #FFF;
margin-bottom: 6px;</string>
       </property>
       <property name="text">
        <string>Schwierigkeit</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox_PvCDifficulty">
       <property name="focusPolicy">
        <enum>Qt::WheelFocus</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: #636363;
font: 75 12pt &quot;Verdana&quot;;
color: #FFF;
height: 35px;
padding: 0px;</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_5">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="styleSheet">
        <string notr="true">font: 75 14pt &quot;Verdana&quot;;
color: #FFF;
margin-bottom: 6px;</string>
       </property>
       <property name="text">
        <string>Wer beginnt?</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox_PvCBeginnt">
       <property name="styleSheet">
        <string notr="true">background-color: #636363;
font: 75 12pt &quot;Verdana&quot;;
color: #FFF;
height: 35px;
padding: 0px;</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QPushButton" name="pushButton_BackPVC">
         <property name="focusPolicy">
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: #424242;
font: 75 12pt &quot;Verdana&quot;;
color: #FFF;
height: 50px;
min-width: 145px;
border-radius: 5px;</string>
         </property>
         <property name="text">
          <string>zurück</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_StartGamePvC">
         <property name="focusPolicy">
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: #424242;
font: 75 12pt &quot;Verdana&quot;;
color: #FFF;
height: 50px;
min-width: 145px;
border-radius: 5px;</string>
         </property>
         <property name="text">
          <string>Spiel starten</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
